app/assets/javascripts/protocols_show.js.coffee

```diff
@@ -4,14 +4,11 @@ $ ->
 
     # Bootstrap Tab persistence
 
```

Comment:
I think this could be a possibility, but wouldn't we need more logic to deal with protocols without services?